### PR TITLE
hostnamectl: add --pretty and --static examples

### PR DESCRIPTION
**Pull request by laurent-indermuehle**
* How to add a pretty name by fixing the static name first
* How to remove a hostname

- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/linux/hostnamectl.md [hunk 1]
```diff
@@ -9,3 +9,11 @@
 - Set the hostname of the computer:
 
 `sudo hostnamectl set-hostname "{{some_hostname}}"`
+
+- Set a pretty hostname for the computer:
+
+`sudo hostnamectl set-hostname --static "{{some_hostname.example.com}}" && sudo hostnamectl set-hostname --pretty "{{some_hostname}}"`
+
+- Reset a hostname to its defaults value by passing the empty string:
```
Comment (bl-ue) [inline pages/linux/hostnamectl.md:17]:
```suggestion
- Reset hostname to its default value:
```

File: pages/linux/hostnamectl.md [hunk 2]
```diff
@@ -9,3 +9,11 @@
 - Set the hostname of the computer:
 
 `sudo hostnamectl set-hostname "{{some_hostname}}"`
```
Comment (bl-ue) [inline pages/linux/hostnamectl.md:11]: I know you didn't do this, but...see my comment below.
```suggestion
`sudo hostnamectl set-hostname "{{hostname}}"`
```

File: pages/linux/hostnamectl.md [hunk 3]
```diff
@@ -9,3 +9,11 @@
...
+
+- Set a pretty hostname for the computer:
+
+`sudo hostnamectl set-hostname --static "{{some_hostname.example.com}}" && sudo hostnamectl set-hostname --pretty "{{some_hostname}}"`
```
Comment (bl-ue) [inline pages/linux/hostnamectl.md:15]: I don't think underscores are actually allowed in domain names...see the [IETF DNS Specification (RFC 1035)](https://tools.ietf.org/html/rfc1035) :)
```suggestion
`sudo hostnamectl set-hostname --static "{{hostname.example.com}}" && sudo hostnamectl set-hostname --pretty "{{hostname}}"`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Thank you for the page update @laurent-indermuehle! I've left some comments below for you to review.

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**